indico/modules/events/sessions/controllers/management/sessions.py

```diff
@@ -50,7 +54,8 @@ def _get_session_list_args(event):
                          subqueryload('blocks').undefer('contribution_count'))
                 .order_by(db.func.lower(Session.title))
                 .all())
-    return {'sessions': sessions, 'default_colors': get_colors()}
+    types = [{'id': int(t.id), 'title': t.name} for t in SessionType.query.with_parent(event).all()]
```

Comment:
Why do you need to cast the id?

Reply:
Shouldn't be needed. And the `.all()` is not necessary; iterating a query does the job already.